src/components/data-table-v2/_data-table-v2-sort.scss

```diff
@@ -9,64 +9,6 @@
 @import '../../globals/scss/vendor/@carbon/elements/scss/import-once/import-once';
 @import '../../globals/scss/vars';
 
-@mixin data-table-v2-sort {
```

Comment:
Data table Sass lives in `data-table` and `data-table-v2` directories, where one imports the other but yet some files look like duplicates of the other? Can all data table styling live in one directory?

Reply:
`_data-table-v2.scss` importing `_data-table.scss` is for keeping backward-compatibility, and need to keep it until `v11`.

Reply:
2 of the v2 data table Sass files don't import the v1 files. They seem to have duplicate content. Can you verify?